src/app/upgrade.cpp

```diff
@@ -37,6 +37,22 @@
 
 namespace
 {
+    void storeLegacyDefaults()
+    {
+        const QVariantHash legacyDefaults {
+            {QLatin1String {"BitTorrent/Session/QueueingSystemEnabled"}, true}
+        };
+
+        SettingsStorage *settingsStorage {SettingsStorage::instance()};
+        for (auto it = legacyDefaults.cbegin(); it != legacyDefaults.cend(); ++it) {
+            const QString &name = it.key();
+            const QVariant &legacyDefault = it.value();
+            const QVariant currentValue = settingsStorage->loadValue(name, legacyDefault);
+            if (currentValue == legacyDefault)
```

Comment:
Really we can omit this checking.

Reply:
Yep. If someone turned it off they'd already have it written to config.